pack.mcmeta

```diff
@@ -1,6 +1,6 @@
 {
   "pack": {
     "pack_format": 5,
```

Comment:
Per issue #69 you may want to update pack_format to "6" (you'll notice this issue if you try older packs with MC 1.16.2 rc 1 or later).